src/TemplateHelpers.hs

```diff
@@ -125,7 +128,18 @@ renderReadme = \case
 renderHtmlDocs :: D.VerifiedPackage -> Text -> Handler (Maybe Html)
 renderHtmlDocs pkg mnString = do
   htmlRenderContext <- getHtmlRenderContext
-  let docsOutput = packageAsHtml (htmlRenderContext pkg) pkg
+
+  depHtmlRenderContexts <- do
```

Comment:
Also I think it would be good to split out some parts of this into a `where` clause e.g. something along the lines of `makeLookupEntry = fmap ((pkgName,) . htmlRenderContext) . either (const Nothing) Just`